cmd/githubgen/allowlist.txt

```diff
@@ -22,3 +21,4 @@ am-kinetica
 mcube8
 sokoide
 zdaratom
+braydonk
```

Comment:
This does not look related to the PR changes, can we move it to a different PR if so? (Also @braydonk happy to sponsor your OTel membership :))

Reply:
yes, it's now in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29727

Reply:
thanks! Resolved

Reply:
@mx-psi I will message you separately on Slack!